pkg/ccl/streamingccl/streamingest/stream_ingestion_test.go

```diff
@@ -100,6 +100,8 @@ func TestStreamIngestionJobWithRandomClient(t *testing.T) {
 	defer close(errCh)
 	_, err := conn.Exec(`SET CLUSTER SETTING bulkio.stream_ingestion.minimum_flush_interval= '0.0005ms'`)
 	require.NoError(t, err)
+	_, err = conn.Exec(`SET enable_experimental_stream_replication = true`)
+	require.NoError(t, err)
```

Comment:
Also a small test here to make sure that the flag is working as expected would also be nice if you have to re-push anyway :)

Reply:
done.